conf/modules/digital_cam_video.xml

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+<module name="video_cam_ctrl" dir="digital_cam">
+  <doc>
+    <description>
+      Digital Camera Triggering of an embedded camera on a linux based autopilot.
```

Comment:
Would be nice to make the first line of the description the name that is supposed to be displayed in the generated modules page. Then the detailed description below in the next line(s).